.sequelizerc

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+module.exports = {
+  'config': path.resolve('database/config', 'config.js'),
+  'models-path': path.resolve('database', 'models'),
+  'seeders-path': path.resolve('database', 'seeders'),
+  'migrations-path': path.resolve('database', 'migrations')
```

Comment:
Because `database` folder is in `src` folder, you should also add the path of src. ex: `src/database`